### ISSUE TITLE
dotnet-install.sh fails to install SDK on Alpine Linux

### ISSUE THREAD

**Issue by natemcmaster** [dotnet/cli#8738]
## Steps to reproduce
```Dockerfile
FROM microsoft/dotnet:2.1-runtime-deps-alpine
WORKDIR /code/build

RUN apk add --no-cache \
        bash \
        wget \
        curl \
        icu-libs \
        openssl

RUN wget https://raw.githubusercontent.com/dotnet/cli/release/2.1.3xx/scripts/obtain/dotnet-install.sh \
    && chmod +x dotnet-install.sh \
    && ./dotnet-install.sh --version 2.1.300-preview2-008251
```

`docker build . -t test`

## Expected  behavior
Download and extract https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.300-preview2-008251/dotnet-sdk-2.1.300-preview2-008251-alpine.3.6-x64.tar.gz

## Actual behavior
```
# ./dotnet-install.sh --version 2.1.300-preview2-008251
dotnet_install: Error: Unable to locate libunwind. Install libunwind to continue
dotnet_install: Error: Unable to locate libssl. Install libssl to continue
dotnet_install: Error: Unable to locate libicu. Install libicu to continue
dotnet_install: Error: Unable to locate libcurl. Install libcurl to continue
```

You can disable pre-req check..
```
export DOTNET_INSTALL_SKIP_PREREQS=1
```
but then this fails because `cp -n` is not supported in Alpine
```
cp: unrecognized option: n
```

**Comment by johnbeisner** [comment]
@natemcmaster
Please use 'https://dot.net/v1/dotnet-install.sh' it is the only supported version.

**Comment by johnbeisner** [comment]
I do not see a "cp -n" in the script. I've checked master, release/2.1.3xx, & the supported version at: 'https://dot.net/v1/dotnet-install.sh'

There is a "cp -R": cp -R $override_switch "$root_path/$path" "$target"
is this what you mean?

**Comment by livarcocc** [comment]
Yes, that should be feasible. @johnbeisner can you work to make sure the change is in the right branch, at which point we should be able to push it.

**Comment by johnbeisner** [comment]
closing: Alpine fix; plus 1 #8821
